eng/scripts/appium-install.ps1

```diff
@@ -40,27 +40,49 @@ Find the script for that on the DevDiv Azure DevOps instance, Engineering team,
 
```

Comment:
See lines just above here. This should ideally match what is on the agents? @rmarinho ?

Reply:
Yes we can ask @mjbond-msft  to update botdeploy script.

Reply:
So my latest changes install the drivers to a local directory, so those versions won't matter to the botdeploy script, but we can ask @mjbond-msft maybe to update the appium version itself (in this case it will now be the latest 2.5.4), since that we are still globally updating.